File: codeforces/contests/Codeforces-Round-1016-Div-3/B.py
for _ in range(int(input())):
    s = input()
    s = s[::-1]
    ans = 0
    for i in range(len(s)):
        if s[i] == '0':
            ans += 1
        else:
            break
    for i in range(ans+1, len(s)):
        if s[i] != '0':
            ans += 1
    print(ans)

File: codeforces/contests/Codeforces-Round-1016-Div-3/B.cpp
// Author: Matin Abbasi
// Date: 2025-04-08
// Description: [Description]

// Includes
#include <bits/stdc++.h>
using namespace std;


// Solve function
void solve(){
  string n;
  cin >> n;
  int ans = 0;
  for (int i = n.length(); i > 0; i--){
    if (n[i] == '0') ans++;
    else break;
  }
  for (int i = n.length() - ans + 1; i > 0; i--){
    if (n[i] != '0') ans++;
  }
  cout << ans << '\n';
}


// Main function
int main() {
  ios::sync_with_stdio(0);
  cin.tie(0);
  int t = 1;
  cin >> t;
  while(t--){
    solve();
  }
  return 0;
}

